python/lsst/ts/mtdome/mock_controller.py

```diff
@@ -794,12 +768,13 @@ async def search_zero_shutter(self) -> float:
 
 async def main() -> None:
     """Main method that gets executed in stand alone mode."""
-    logging.info("main method")
+    log = logging.getLogger("MockMTDomeController")
+    log.info("main method")
     # An arbitrarily chosen port. Nothing special about it.
     port = 5000
-    logging.info("Constructing mock controller.")
-    mock_ctrl = MockMTDomeController(port=port)
-    logging.info("Starting mock MTDome controller.")
+    log.info("Constructing mock controller.")
+    mock_ctrl = MockMTDomeController(host=tcpip.DEFAULT_LOCALHOST, port=port, log=log)
```

Comment:
I gently suggest making MockMTDomeController have a default host of tcpip.DEFAULT_LOCALHOST so you don't have to specify it here.

You could even eliminate the host constructor argument in MockMTDomeController (I would probably do that).

Reply:
It turns out that the `host` constructor argument is mandatory because of the `create_server` method in `tcpip.BaseOneClientServerTestCase` so I kept it and set it by default to `tcpip.DEFAULT_HOST` but I eliminated the constructor argument everywhere where a `MockMTDomeController` is constructed.

Reply:
Good point, but: I actually fixed it as part of ts_tcpip 1.1.1 -- though as a commit that I had not pushed when you reviewed it (oops). It's a really small change: if the server class is OneClientServer and create_server's `**kwargs` does not include `host` then add it. So usually there will be no host argument.

I discovered it in ts_atdome, and when I fixed it I did not need to update ts_atdome for ts_tcpip 1.1.1.

Feel free to leave the host argument; I just wanted you to know that it is not actually required any more.

Reply:
I indeed missed that. I've removed the host argument now.